manifest-lock.overrides.yaml

```
@@ -8,4 +8,10 @@
 # in the `metadata.reason` key, though it's acceptable to omit a `reason`
 # for FCOS-specific packages (ignition, afterburn, etc.).
 
-packages: {}
+packages:
+  skopeo:
+    evr: 1.5.1-1.fc35
```

Ouch, looks like it was also missing an epoch:

```suggestion
    evr: 1:1.5.1-1.fc35
```